platform_versioned_docs/version-23.4.0/data/data-studios.mdx

```diff
@@ -23,11 +23,27 @@ Before you get started, ensure you have the following:
 Currently, Data Studios only supports AWS Batch compute environments that **do not** have Fargate enabled.
 :::
 
-### Overview
+## Overview
 
 Select the **Data Studios** tab to view all data studios. The list includes the name, cloud provider, analysis template, region, creator, creation date, and the status of each session. In this view, you can add a new data studio, as well as start, stop, and connect to an existing data studio. You can dynamically filter the list of data studio sessions using the search bar.
 
-### Add a data studio
+## Data studio checkpoints
+
+When you start a data studio, it automatically creates a *checkpoint*. A checkpoint persists changes that you make to the data studio root filesystem. The checkpoint is persisted on the compute environment's working directory.
+
+If you start a data studio from a previously created checkpoint, changes such as installed software packages and configuration files are restored and available in the data studio session. Changes to mounted datasets are not included in the checkpoint.
```

Comment:
```suggestion
When you restart, or start a new, data studio from a previously created checkpoint, changes such as installed software packages and configuration files are restored and available in the data studio session. Changes to mounted datasets are not included in the checkpoint.
```

Reply:
@robnewman, this was that attempt at concrete examples.